lib/restforce/db.rb

```diff
@@ -140,6 +144,28 @@ def self.reset!
       @last_run = nil
     end
 
+    # Public: Get the hashed version of the passed salesforce ID. This will
+    # converts 15-digit Salesforce IDs to their corresponding 18-digit version.
+    # Returns any passed 18-digit IDs back, untouched.
+    #
+    # Returns a String.
+    # Raises an ArgumentError if the passed String is not 15 characters.
+    def self.hash(salesforce_id)
```

Comment:
Can we use a more descriptive name than `hash`?

Other than that, this PR looks grand.